assets/app/views/project.html

```diff
@@ -2,7 +2,7 @@
   <project-page>
     <div ng-controller="OverviewController">
       <tasks></tasks>
-      <h1>Project {{project.displayName || project.metadata.name}}</h1>
+      <h1>Project {{project | displayName || project.metadata.name}}</h1>
```

Comment:
there isn't a shortcut filter for the displayName annotation yet like there is for description, you would need to add one.  See the example for description:

https://github.com/openshift/origin/blob/master/assets/app/scripts/filters/resources.js#L24